app/views/users/attendees.html.erb

```diff
@@ -0,0 +1,18 @@
+
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @user.each do |users| %>
```

Comment:
Should be `|user|` since you are using `user` on https://github.com/lightcastle/becamp_rails/pull/1/files#diff-d01b47eca688d86d23e89a2ebe443dddR13